docs/source/index.rst

```diff
@@ -96,6 +96,7 @@ PyTorch Lightning Documentation
    cloud_training
    amp
    slurm
+   cluster
    child_modules
```

Comment:
just an idea, we could combine the slurm and cluster navigation items into one and make instead two sections, one for slurm and one for non-slurm